backend/uclapi/dashboard/webhook_views.py

```diff
@@ -0,0 +1,187 @@
+from django.http import JsonResponse
+from .models import App, User
+import keen
+from django.core.exceptions import ObjectDoesNotExist
+import requests
+from .app_helpers import generate_secret, is_url_safe
+
+
+def user_owns_app(user_id, app_id):
+    user = User.objects.get(id=user_id)
+    try:
+        app = App.objects.get(id=app_id)
+    except ObjectDoesNotExist:
```

Comment:
Prefer `App.DoesNotExist` here for clarity, but I'm just being picky lol

Reply:
Agreed, that's much more beautiful and allows us to remove one line of code 💪